### DISCUSSION TITLE
Buttons doesn't work 'send() get an unexpected parameter view'

### DISCUSSION THREAD

**Discussion by edujime23** [nextcord/nextcord#335] [Q&A] [answered]
At first everything was fine and correct, but out of nowhere it stopped working. I reinstalled nextcord and they worked again, but now they stopped working again and reinstalling it from nextcord no longer works, what do I do?

`class View(nextcord.ui.View):
        def __init__(self):
            super().__init__(timeout=None)

        @nextcord.ui.button(label="+10",style=nextcord.ButtonStyle.blurple)
        async def add__(self,button : nextcord.Button , interaction : nextcord.Interaction):
            Vol = float(ctx.voice_client.source.volume)
                
            if (Vol * 100 + 10) > 151:
                return await ctx.send("Volume too high")
            Vol += float(10 / 100)
  
            ctx.voice_client.source.volume = Vol
      
            await interaction.message.edit(content=f"Changed volume to {round(Vol * 100)}%")
             
                    
        @nextcord.ui.button(label="-10",style=nextcord.ButtonStyle.blurple)
        async def rest__(self,button : nextcord.Button , interaction : nextcord.Interaction):
            Vol = float(ctx.voice_client.source.volume)
            
            if (Vol * 100 - 10) < -1:
                return await interaction.response.send_message("Volume too low")
            Vol -= float(10 / 100)
                 ctx.voice_client.source.volume = Vol
             await interaction.message.edit(content=f"Changed volume to {round(Vol * 100)}%")
   
await ctx.send(f"Changed volume to {round(Vol)}%",view=View())`

**Comment by TAG-Epic** [comment]
Please give more info like your nextcord version, full traceback, and the code causing the error.

**Comment by DenverCoder1** [comment, marked as answer]
Make sure to uninstall discord.py or similar libs since it may be using the wrong methods. In Nextcord, send should always support views.

Also, where is ctx defined? To use it in a view, it should be made an attribute of the class.
